app/src/main/java/com/smlnskgmail/jaman/hashcheckerlite/logic/settings/api/SettingsHelper.java

```diff
@@ -41,4 +43,7 @@
 
     void setRefreshSelectedFileStatus(boolean status);
 
+    boolean canShowRateAppDialog(Context context);
```

Comment:
Delete `Context` argument from `canShowRateAppDialog` and `increaseHashGenerationCount` methods. Context available in interface realization (`SharedPreferencesSettingsHelper`).

Reply:
okay!